### PR TITLE
24 sprint 2 view single student information

### PR DESCRIPTION
**Pull request by ipr17**
Ok so key things I changed (all on /student):

Added 'Remove Student' button, which deletes the attendance record for that student. 
Added the ability to edit sign in/out times.
Added ability to sign out individual students (but only from student page - home page coming soon).
Now pressing 'Save Changes' will cause updates to all fields where data was given.

### CODE REVIEW

**Review by DeltaO3 — CHANGES_REQUESTED** [review]
looks great! code mostly looks ok:

Some issues tho:
You arent checking nonetype, causing a crash (this is an edge case but still crashes)
image
Error message on page could potentially display what the issue was? currently just gives "error updating student details"

Not sure why  sign in includes seconds but sign out doesnt? can this be fixed?
image
Seems to be causing issues - cant sign students out, keeps printing that its an invalid format?

Displays correctly if i sign all students out first

**Comment by ipr17** [comment]
@DeltaO3 I cannot replicate the case where the seconds do not show up for sign out time, do you remember how you came across this? I also have not gotten any errors about signing students out, so I don't know how to debug that either.

**Review by DeltaO3 — APPROVED** [review]
LGTM!

**Comment by ipr17** [comment]
make it so that if a student removes or adds consent, it changes their student value

**Review by livvy10 — APPROVED** [review]
requested changes will be done on a separate issue

File: app/routes.py [hunk 1]
```diff
@@ -295,11 +295,18 @@ def delete_database(response):
 # STUDENT - /student/
```
Comment (DeltaO3) [inline app/routes.py:295]: Should this not have login required?

**Comment by livvy10** [comment]
are we happy with it as a stringfield? I like it more as textarea as it has more space and you can do returns and things, but not a big issue at all
also it's coming up with None as the comment if nothing is set, but i think it should be blank
either I think we could initialise all attendance entries with "" as comment

**Review by livvy10 — APPROVED** [review]

**Pull request merged**